source/blog/2021-03-29-dbal-2.13.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "New Release: Doctrine DBAL 2.13 with Deprecations and Forward Compatibility"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2021/03/29/dbal-2.13.html
+---
+
+We have released the last minor version in the 2.x family of Doctrine DBAL with
+additional forward compatibilty to DBAL 3 around `Statement` and `Result` classes
```

Comment:
Technically, there are both classes and `interfaces` named `Statement` and `Result` that this article refers to. I'd be less specific and refer to them as APIs.